### DISCUSSION TITLE
Has the rain template been dropped?

### DISCUSSION THREAD

**Discussion by kwee27** [primefaces/community#1526] [PrimeFaces Templates] [answered]
Looking for the link to its documentation and now can't find any reference to the rain template anywhere on the primefaces site.

**Comment by rider87** [comment]
One month later... did you got an answer from Primetek? I thought they are monitoring also the forum?

**Comment by onursenture** [comment, marked as answer]
Hello, you can access the official statement on this matter at the following address: https://github.com/orgs/primefaces/discussions/1806

**Comment by tandraschko** [comment]
we decided to only maintain
          - "apollo"
          - "atlantis"
          - "avalon"
          - "diamond"
          - "freya"
          - "poseidon"
          - "ultima"
          - "verona"
          
for now
